commit 1064a53e4ff357dc649a8c4a0a41dfb5a1191bba
Author: Karthikeyan Krishnasamy <karthikeyan@linumiz.com>
Date:   Sat Mar 23 13:39:55 2024 +0530

    sys-utils: hwclock-rtc: fix pointer usage
    
    passing double pointer doesn't fill param value
    
    Signed-off-by: Karthikeyan Krishnasamy <karthikeyan@linumiz.com>

diff --git a/sys-utils/hwclock-rtc.c b/sys-utils/hwclock-rtc.c
index 79867f64f..113d4e8ca 100644
--- a/sys-utils/hwclock-rtc.c
+++ b/sys-utils/hwclock-rtc.c
@@ -429,7 +429,7 @@ static int resolve_rtc_param_alias(const char *alias, __u64 *value)
 /* kernel uapi __u64 can be defined differently than uint64_t */
 static int strtoku64(const char *str, __u64 *num, int base)
 {
-	return ul_strtou64(str, (uint64_t *) &num, base);
+	return ul_strtou64(str, (uint64_t *) num, base);
 }
 
 /*
